app/controllers/photos_controller.rb

```diff
@@ -0,0 +1,24 @@
+class PhotosController < ApplicationController
+	def index
+		@photos = Photo.all.sort_by { |p| p.created_at }.reverse
```

Comment:
Style/SymbolProc: Pass `&:created_at` as an argument to `sort_by` instead of a block.